include/kframework/gas.md

```diff
@@ -0,0 +1,55 @@
+KEVM Gas
+========
+
+Here is a representation of `Gas`. This sort is used by `<gas>`, `<callGas>`, and `<gasUsed>` cells of the EVM configuration. As is in this file, `Gas` is a super-sort of `Int` with no differing behaviour. However the distinction at this level allows for an extension of infinite gas later, see `infinite-gas.md`.
+
+```k
+module GAS-SYNTAX
+    imports INT-SYNTAX
+    
+    syntax Gas ::= Int
+    
+    syntax Gas ::= Gas "*Gas" Gas [function, left, total]
+                 | Gas "/Gas" Gas [function, left]
+                 > left:
+                   Gas "+Gas" Gas [function, left, total]
+                 | Gas "-Gas" Gas [function, left, total]
+
+    syntax Bool ::= Gas  "<Gas" Gas [function, left, total]
+                  | Gas "<=Gas" Gas [function, left, total]
+                  | Gas  ">Gas" Gas [function, left, total]
+                  | Gas ">=Gas" Gas [function, left, total]
```

Comment:
I don't see `>Gas` or `>=Gas` actually used anywhere in the semantics, we just are providing lemmas for it. Can we just remove these two variants altogether?

Reply:
I thought for completeness into the future, should someone wish to use the operator it would be there. But I don't think it is necessary.

Reply:
We always, by convention, use the `<` variants, because it reduces how many lemmas we have to write. So not including the other variants (for `>`), is a way to enforce that convention on future users.